src/main/python/systemds/operation_node.py

```diff
@@ -0,0 +1,218 @@
+
+    def l2svm(self, labels: DAGNode, **kwargs) -> OperationNode:
+        """
+        Perform l2svm on matrix with labels given.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        params_dict = {'X': self, 'Y': labels}
+        params_dict.update(kwargs)
+        return OperationNode('l2svm', named_input_nodes=params_dict)
+
+    def sum(self) -> OperationNode:
+        """
+        Calculate sum of matrix.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('sum', [self], output_type=OutputType.DOUBLE)
+
+    def col_sums(self) -> OperationNode:
+        """
+        Calculate sum of cols.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('colSums', [self])
+
+    def row_sums(self) -> OperationNode:
+        """
+        Calculate sum of rows.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('rowSums', [self])
+
+    def mean(self) -> OperationNode:
+        """
+        Calculate mean of matrix.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('mean', [self], output_type=OutputType.DOUBLE)
+
+    def col_means(self) -> OperationNode:
+        """
+        Calculate means of cols.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('colMeans', [self])
+
+    def row_means(self) -> OperationNode:
+        """
+        Calculate means of rows.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('rowMeans', [self])
+
+    def var(self) -> OperationNode:
+        """
+        Calculate variance of matrix.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('var', [self], output_type=OutputType.DOUBLE)
+
+    def col_vars(self) -> OperationNode:
+        """
+        Calculate variances of cols.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('colVars', [self])
+
+    def row_vars(self) -> OperationNode:
+        """
+        Calculate variances of rows.
+        :return: `OperationNode` representing operation
+        """
+        self._check_matrix_op()
+        return OperationNode('rowVars', [self])
+
+    def abs(self) -> OperationNode:
+        """
+        Calculate absolute.
+        :return: `OperationNode` representing operation
+        """
+        return OperationNode('abs', [self])
+
+    def moment(self, moment, weights: DAGNode = None) -> OperationNode:
+        # TODO write tests
```

Comment:
why is this comment only here?

Reply:
because this function is the only one that is still missing tests.